app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
> I see the webhooks as something useful for integrating Portus/distribution into a workflow.

I'm absolutely with you. This is actually my use case ;-)

> Right now we don't see the need to forward events triggered from portus

Fine with me as well.

What I'm concerned is

> Right now we want to focus on *notifications about new repositories/tags being created*. In the near future we will support about notifications for delete events

Why do you want to filter out delete events? Why don't we pass _all_ notifcations from registry regarding the namespace to the regsitered endpoint?